Pkgfile

```diff
@@ -9,8 +9,8 @@ vars:
   arm_trusted_firmware_sha512: 958cf8f9e258638e59d0fbd8b053fce7d8a9ea2fc922686c9d20ea16f79f55219ac18a12ab240c528ee98e49c2e0eef4c963fdb255cc14b92437a5b3cffc8640
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
-  uboot_version: 2023.01
-  uboot_sha256: 69423bad380f89a0916636e89e6dcbd2e4512d584308d922d1039d1e4331950f
-  uboot_sha512: 417a28267eb7875820d08fafc7316f164663609378637539e71648b0b9b7d28796b6c381717f31b0ab6472805fefd32628ef7d1b2e7b9f3c51c8ad122993f679
+  uboot_version: 2024.01
```

Comment:
i think this will break the other builds since it has patches

Reply:
I'm reviewing this point and I'll get back to you

Reply:
Yes, the only patches are for enabling the SPI chip support. Most installs happen on SD cards or eMMC modules; not all boards are sold with pre-soldered SPI chips. 

Dropping support for the SPI flash build will ease the process while impacting few users.

Reply:
I was thinking of dropping it in a different PR and clean up the docs, I don't mind dropping here also

Reply:
I've dropped all patches here https://github.com/siderolabs/sbc-rockchip/pull/6

Reply:
Perfect! I'll rebase this branch after merging the new PR. Thanks!